react/SocialNetworks.tsx

```diff
@@ -15,31 +15,34 @@ interface Props {
   showInColor: boolean
 }
 
-const SocialNetworks: React.FC<Props> = ({ title, socialNetworks }) => {
+const CSS_HANDLES = ['socialNetworksTitle', 'socialNetworksContainer']
+
+const SocialNetworks: StorefrontFunctionComponent<Props> = ({
+  title,
+  socialNetworks = [],
+}) => {
+  const handles = useCssHandles(CSS_HANDLES)
+
   return (
-    <div>
+    <Fragment>
```

Comment:
Why that change? I would keep the div just for the sake of it.

Reply:
No need did that just to clean-up the tree. But I was not sure if it was a good idea because that could break someone who depended on that `<div>` for the layout...
I'll keep the div and add a CSS handle to it hahaha